TLM/TLM/UI/MainMenu/VersionLabel.cs

```diff
@@ -9,6 +10,13 @@ public class VersionLabel : UILabel {
                 MainMenuPanel.SIZE_PROFILES[0].MENU_WIDTH,
                 MainMenuPanel.SIZE_PROFILES[0].TOP_BORDER);
             text = TrafficManagerMod.ModName;
+
+            if(LoadingExtension.Instance.HotReload) {
+                // make it easier to Identify Hot reload.
+                text += " HOT RELOAD " +
```

Comment:
Would users ever see this? For example, they are subscribed to mod, and have game sitting at Main Menu screen. We push update to mod, Steam downloads it (replacing files on disk with new version).

Or does hotloading only happen if mod files are in `...\Colossal Order\Cities_Skylines\Addons\Mods` folder?

Reply:
Changing the assemblies while at main menu does not show hot reload text. only when changing assemblies while in game, hot reload text will appear.

users can see this if they replace their assemblies ... somehow.

`LoadingExtension.Instance.HotReload = InGame()` at `LoadingExtension.OnCreate()` which is only possible while swapping DLLs while in game.

Reply:
> users can see this if they replace their assemblies ... somehow.

Steam client won't update anything while in-game, so it can be used by modders - manually/script